### DISCUSSION TITLE
[REQUEST] Sound feature

### DISCUSSION THREAD

**Discussion by wingcwf** [Zrips/CMI#1168] [Feature Requests]
### Description of issue or feature **request:**

WarpGui
Example: ( https://gyazo.com/a1b8c819bb72d89955f031d9af02ec65 )
1. Add Sound when open ' warpgui / successful tp / fail tp(no permission)
`Players can choose to turn the sound on or off in config`
2. Add Icon used when player no permission/unavailable (Like kitedit)
3. Add Info to customize ( https://pastebin.com/LV98KBC3 )
4. Maybe add particles when open warpgui? ( unimportant )
Example: ( https://gyazo.com/6c945bcea8600ff4a814c45b9b373136 )

KitsGui
Example: ( https://gyazo.com/fab003fc4e729a82fe5164b2f7a75720 )
1. Add Sound when open ' kitsgui / successful get kit / fail (no permission)
`Players can choose to turn the sound on or off in config`
2. Add Icon used when kit in cooldown
3. Add Info to customize ( https://pastebin.com/aSRuCjpg )
4. Add Players can see all kits list even if they don't have permission. 
5. Maybe add particles when open kitsgui? ( unimportant )
Example: ( https://gyazo.com/6c945bcea8600ff4a814c45b9b373136 )

RTP / home :
1. Add Successful Tp Sound

Scheduler Announcer:
1. Add Sound
2. Add Multi-line ```\n . |```
3. Add JSON messages

**Comment by mercurialmusic** [comment]
The join server sound and teleport sounds are already possible with CMI's eventCommands.yml

Also: https://www.spigotmc.org/resources/bottledexp.2815/ :)

**Comment by wingcwf** [comment]
@Zrips
Can you add it to priority processing?
